Python/import.c

```diff
@@ -2240,11 +2259,17 @@ init_importlib(PyThreadState *tstate, PyObject *sysmod)
     if (PyImport_ImportFrozenModule("_frozen_importlib") <= 0) {
         return -1;
     }
-    PyObject *importlib = PyImport_AddModule("_frozen_importlib"); // borrowed
+
+    PyObject *name = PyUnicode_FromString("_frozen_importlib");
+    if (name == NULL) {
+        return -1;
+    }
+    PyObject *importlib = import_add_module(tstate, name);
+    Py_DECREF(name);
```

Comment:
Why not use `PyImport_AddModuleRef` here also?

Reply:
Right, I did that.